packages/builders/LICENSE

```diff
@@ -0,0 +1,22 @@
+(The MIT License)
+
+Copyright (c) 2017 Narwhal Technologies
```

Comment:
2018. Other licence files should be updated as well?

Reply:
Yeah, should we just copy the LICENSE file from the root when we publish?